data-raw/classifiers.R

```diff
@@ -0,0 +1,37 @@
+#!/usr/env Rscript
+
+# This script prepares classifier objects used in the package:
+# classifier.ktsp_unweighted
+# classifier.rf_weighted
+# To run this script, you must have appropriate credentials set up
+# to access the private S3 bucket s3://data-lab-mb-ssp
+
+library(optparse)
+option_list <- list(
+  make_option(
+    c("-p", "--aws_profile"),
+    type = "character",
+    default = "",
+    help = "Name of AWS profile to use for syncing baseline.rds"
+  )
+)
+opt <- parse_args(OptionParser(option_list = option_list))
+
+# Download file
+temp_file <- tempfile()
+s3_path <- "s3://data-lab-mb-ssp/models"
+sync_call <- glue::glue("aws s3 sync '{s3_path}' '{temp_file}' --exclude '*' --include 'baseline.rds' --exact-timestamps --profile {opt$aws_profile}")
```

Comment:
```suggestion
sync_call <- glue::glue("aws s3 sync '{s3_path}' '{temp_dir}' --exclude '*' --include 'baseline.rds' --exact-timestamps --profile {opt$aws_profile}")
```
If you take my previous suggestion.